### ISSUE TITLE
modify the heuristics

### ISSUE THREAD

**Issue by joelgeorgem** [jp-praveen/Rendezvous#21]
https://github.com/jp-praveen/Rendezvous/blob/d5f87457cd18d54b07a22c59b95f45330363c2c0/realistic_position.m#L39

Make heuristic_cost = dvmax/sequence_length

**Comment by jp-praveen** [comment]
Yes Sir

**Comment by joelgeorgem** [comment]
Also consider, yet another heuristic in which

heuristic_cost(i) = dvmax/[outdegree](https://in.mathworks.com/help/matlab/ref/digraph.outdegree.html)(i)

**Comment by jp-praveen** [comment]
Yes Sir, So we have two different approaches under Heuristic itself and totally 3 approaches, 1 greedy and 2 heuristics

**Comment by joelgeorgem** [comment]
Think about other heuristics too.

For example, consider a heuristic with 

heuristic_cost = dv corresponding to the transfer from sequence_heuristic(2) to sequence_heuristic(3)

**Comment by joelgeorgem** [comment]
That is like two-step greedy rather than immediate greedy!

**Comment by jp-praveen** [comment]
Yes sir, I will come up with other heuristics before meeting you on Wednesday.
Should different heuristics cost be added together and compared or each is like a new approach?

**Comment by joelgeorgem** [comment]
Each is a different algorithm.

**Comment by jp-praveen** [comment]
Yes Sir, I will do the changes and will notify you.

**Comment by jp-praveen** [comment]
Dear Sir,
         I did the tasks which we discussed over the phone. Shall I meet you at 10 AM tomorrow?

**Comment by jp-praveen** [comment]
The final code is in the optimal_debris_removal branch and is named as runcode.m

**Comment by joelgeorgem** [comment]
Come at 9:50 AM itself; I will meet you immediately after my 9 AM class.

**Comment by jp-praveen** [comment]
Yes Sir. I will be there at 9:50 AM.
